devdocs/orientation-guide.md

```diff
@@ -0,0 +1,120 @@
+![](images/teammates_heading.png)
+
+# **Contributor Orientation Guide**
+
+We welcome contributions from anyone, in particular, students (see [here](https://teammatesv4.appspot.com/about.html) for a list of our contributors). One of the main objectives of TEAMMATES is to help students get experience in a OSS production environment. Here are some information that might be useful to would-be contributors.
+
+
+
+[Knowledge required](#h.abftcg92xk1k)
+
+[Workload](#h.8yn20iebwilp)
+
+[Getting started](#h.4es3v3yquxal)
+
+[Contacting us](#h.5lngkuxtl5mo)
```

Comment:
I think the link over here should be the section headings, all converted to lower case and hyphenated for spaces. eg: 
`[Knowledge required](#knowledge-required)`